libs/sdk-ui-charts/src/charts/headline/CoreHeadline.tsx

```diff
@@ -9,16 +9,24 @@ import {
     withEntireDataView,
 } from "@gooddata/sdk-ui";
 import { ICoreChartProps } from "../../interfaces/index.js";
-import HeadlineTransformation from "./internal/HeadlineTransformation.js";
 import { defaultCoreChartProps } from "../_commons/defaultProps.js";
+import { IHeadlineTransformationProps } from "./HeadlineProvider.js";
 
-type Props = ICoreChartProps & ILoadingInjectedProps;
-export class HeadlineStateless extends React.Component<Props> {
+/**
+ * @internal
+ */
+interface ICoreHeadlineExtendedProps {
+    headlineTransformation: React.ComponentType<IHeadlineTransformationProps>;
```

Comment:
As far as I know, the CoreHeadline component is tagged as an internal component, which means our expectation is that it will only be used for the Headline component and PluggableHeadline. Therefore, it probably won't break backward compatibility. If you have any ideas about cases that could potentially lead to breakage, please let me know.

Thank you so much!